src/AndorSIF.jl

```diff
@@ -197,7 +197,7 @@ function FileIO.load(fs::Stream{format"AndorSIF"})
 
     offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
 
-    pixelmatrix = Array{Gray{Float32}}(undef, height, width, frames)
+    pixelmatrix = Array{Gray{Float32}}(undef, width, height, frames)
```

Comment:
After you've read it, consider returning `ImageMeta(PermutedDimsArray(pixelmatrix, (2, 1, 3)), properties)` so the orientation matches the standard way of viewing. (You can check with ImageView.)

Reply:
You might also consider renaming this variable `pixelarray` since it's not 2d.

Reply:
It’ll be nice if backward compatibility is maintained for images that did work correctly though. This would be especially important for square images since it’ll most likely result in a silent error.